PlayRho/Common/RealNum.hpp

```diff
@@ -73,6 +73,10 @@ namespace playrho
     ///
     constexpr auto Pi = Real(3.14159265358979323846264338327950288);
 
+    /// @brief SquareRootTwo.
+    ///
+    constexpr auto SquareRootTwo = Real(1.414213562373095048801688724209698078569671875376948073176679737990732478462);
+
```

Comment:
Unfortunately adding `SquareRootTwo` to this file won't work unless it's also added to the `RealNum.hpp.in` file. `*.in` files should be files that override their namesakes by `cmake` (or any other such facility). And loading override files with anything other than what's being set at build time, is less appealing to me than the constant being in the `Settings.hpp` file.